cirq-google/cirq_google/calibration/phased_fsim.py

```diff
@@ -996,6 +997,12 @@ def with_zeta_chi_gamma_compensated(
             (cirq.rz(0.5 * gamma - beta).on(a), cirq.rz(0.5 * gamma + beta).on(b)),
         )
 
+    def _unitary_(self) -> np.array:
```

Comment:
Could you add a docstring to mention that this is cirq unitary protocol implementation?